static/js/containers/CollectionDetailPage.js

```diff
@@ -157,20 +157,12 @@ class CollectionDetailPage extends React.Component {
     </div>;
   }
 
-  renderError = (collectionError: Object) => {
-    return <h2 className="mdc-typography--title error">
-      { collectionError.detail }
-    </h2>;
-  };
-
   render() {
-    const { collection, collectionError } = this.props;
+    const { collection } = this.props;
```

Comment:
Can you also remove `collectionError` from the props of this component if it's unused?

Reply:
OK

Reply:
It is still potentially used in `componentDidUpdate`